footprint/run_suite.py

```
@@ -0,0 +1,170 @@
+        if os.path.isfile(path):
+            return path
+    raise RuntimeError(
+        "llvm-size not found; install with `rustup component add llvm-tools`"
+    )
+
+
+SIZE_TOOL = find_llvm_size()
+
+
+def run_cmd(args, cwd=None, timeout=TIMEOUT_RUN):
+    return subprocess.run(args, capture_output=True, text=True, timeout=timeout, cwd=cwd)
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)